src/main/java/edu/harvard/hul/ois/fits/AESModel.java

```diff
@@ -197,20 +197,20 @@ private double timeUnitToAddress(String time) {
         // hours:minutes:seconds:milliseconds
         if (parts.length >= 3) {
             // hours
-            seconds = (Long.valueOf(parts[0]) * 60 * 60);
+            seconds = (Long.parseLong(parts[0]) * 60 * 60);
```

Comment:
Is there a performance or other benefit of using `Long.parseLong` instead of `Long.valueOf`. Based on the [javadocs](https://docs.oracle.com/javase/7/docs/api/java/lang/Long.html#parseLong(java.lang.String)), the benefit is not clear.

Reply:
Yeah, the diff is that `valueOf` creates a Long, which is then immediately unboxed to a `long` based on the usage